src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -354,17 +411,48 @@ class DemuxFastqs
   }
 
   override def execute(): Unit = {
+    val sortProgress:  ProgressLogger = ProgressLogger(logger, verb="Sorted", unit=2e6.toInt)
+    val writeProgress: ProgressLogger = ProgressLogger(logger, verb="Wrote ", unit=2e6.toInt)
+
     Io.mkdirs(this.output)
     Io.assertCanWriteFile(this.metricsPath)
 
+    // Some logging information about async reading and writing
+    {
+      val readerMessage = this.asyncOptions.asyncReadingType match {
+        case DemuxFastqsAsyncReadingType.NoAsync         => "not use async reading"
```

Comment:
Since this enum exists purely for specifying options here, why not just put these strings inside the case objects?